core/messaging/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@baseline-protocol/messaging",
+  "version": "0.0.2",
+  "description": "Baseline core messaging",
+  "license": "CC0 1.0 Universal",
+  "main": "./dist/cjs/index.js",
+  "types": "./dist/cjs/index.d.ts",
+  "unpkg": "./dist/umd/index.min.js",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build:cjs": "./node_modules/.bin/tsc -p tsconfig.cjs.json",
+    "build:umd": "webpack",
+    "build": "yarn clean && yarn build:cjs && yarn build:umd",
+    "clean": "rm -rf ./dist",
+    "geth:start": "npm run geth:stop 2&>/dev/null && docker run -td -p 8546:8546 ethereum/client-go --shh --ws --wsaddr 0.0.0.0 --wsorigins '*' --wsapi=web3,shh,net --testnet --nodiscover --maxpeers 0 && sleep 2",
```

Comment:
Should we force a version of the geth container since they announced that future versions will no longer container whisper code?

Reply:
👍 

Either that or drop whisper support all together :/

Reply:
is there anyway we can limit or specify a config var for whisper and corresponding dependence on geth? @bitwiseguy @kthomas